daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md

```diff
@@ -293,6 +293,19 @@ auth:
   secretStore: <SECRET_STORE_NAME>
 ```
 
+## Sending and receiving multiple messages
+
+Apache Kafka component natively supports sending and receiving multiple messages in a single operation (Bulk Pubsub feature).
+
+### Configuring bulk subscribe
+
+When subscribing to a topic, you can configure `bulkSubscribe` options. Refer to [Subscription methods]({{< ref subscription-methods >}}) for more details. (TODO: Add link to bulk subscribe docs)
+
+| Configuration | Default |
+|----------|---------|
+| `maxBulkAwaitDurationMs` | `10000` (10s) |
```

Comment:
Same comment as the Azure Service bus docs pR #3054 

These fields needs to be added to the component spec example at the start as well as into the table of values. Putting them here does not make them discoverable. The table is the source of truth for the spec.

Reply:
@msfussell This is not a component specific spec. This is a bulk Subscribe specific spec. 
That is why I added a TBD. This has to be changed after https://github.com/dapr/docs/pull/3077 PR is merged. Example config in that PR https://github.com/dapr/docs/pull/3077/files#diff-081f2bf4c37527e63a2ebbfc0ca6b08ee9606a6650d84c54dc2b374d61dd85e2R170-R186